AUDIT   2022/11/04 13:35:54 [Workflow:workflow] Starting workflow (Writing log to log/scipipe-20221104-133554-workflow.log)
AUDIT   2022/11/04 13:35:54 [Task:index] Output file already exists, so skipping: ref.mmi
AUDIT   2022/11/04 13:35:54 [Task:alignment] Output file already exists, so skipping: output/alignment.sam
AUDIT   2022/11/04 13:35:54 [Task:samtools] Output file already exists, so skipping: output/result.txt
AUDIT   2022/11/04 13:35:54 [Task:parse] Executing: ls -a | python3 input/parser.py ../output/result.txt > output/parsed.txt
ERROR   2022/11/04 13:35:54 [Task:parse] Command failed!
Command:
ls -a | python3 input/parser.py ../output/result.txt > output/parsed.txt

Output:
python3: can't open file 'input/parser.py': [Errno 2] No such file or directory

Original error:exit status 2


